### ISSUE TITLE
Order metrics shown on tooltip

### ISSUE THREAD

**Issue by macobo** [PostHog/posthog#2620]
## Is your feature request related to a problem?

This tooltip is hard to follow:

![Screenshot from 2020-12-02 10-53-19](https://user-images.githubusercontent.com/148820/100850552-a304d200-348c-11eb-8535-88804626918e.png)

## Describe the solution you'd like

Order the tooltip by volume somehow.

One potential ordering - take the last data point, order by that.

## Additional context

Not sure why versions which are at 0 every datapoint even show up.

Came up during #2560

**Comment by macobo** [comment]
> Not sure why versions which are at 0 every datapoint even show up.

This is coming from the backend - `TOP_ELEMENTS_ARRAY_OF_KEY_SQL` is returning all possible values in time frame, ignoring all other filters.

@EDsCODE would it be reasonable to add filter clause to `TOP_ELEMENTS_ARRAY_OF_KEY_SQL` as well? 

Also, should we also sort the result from backend to solve this issue?

**Comment by EDsCODE** [comment]
Yeah, I think we should add filters clause to that SQL and for `TOP_PERSON_PROPS_ARRAY_OF_KEY_SQL`

**Comment by paolodamico** [comment]
Relevant user feedback ([Slack](https://posthogusers.slack.com/archives/CT7HXDEG3/p1607017725135900)), optionally hide `0` or `null` values.

**Comment by Twixes** [comment]
This is solved in the current production tooltips BUT not in the latest `line-graph-v2` iteration @alexkim205
